server/router.go

```diff
@@ -63,7 +63,7 @@ func (s *Server) Serve() error {
 		signal.Notify(c, os.Interrupt, os.Signal(syscall.SIGTERM))
 		sig := <-c
 		s.l.InfoD("shutdown-initiated", logger.M{"signal": sig.String()})
-		ctx, cancel := context.WithTimeout(context.Background(), 30*time.Second)
+		ctx, cancel := context.WithTimeout(context.Background(), 65*time.Second)
```

Comment:
This should remain the same because it is just a timeout for the `server.Shutdown` call. We want a `time.Sleep(65*time.Second)` before this line